test/Microsoft.AspNet.Hosting.Tests/Fakes/StartupBase.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Builder;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Fallback;
```

Comment:
Don't need this namespace. Might be a few other extraneous namespaces as well (Builder?).